CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0
```

Comment:
We typically try to commit `CHANGELOG` changes as part of a commit that occurs after the PR merge. For example:

- https://github.com/azavea/terraform-aws-ecs-cluster/commit/292a5bebfac03734432a6683431d5898cbfc691e
- https://github.com/azavea/terraform-aws-ecs-cluster/commit/c3e36484709f1034c2705cb57548f6ee3445a7c4